packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -7,8 +7,58 @@ import { Canvas, Title } from '@storybook/addon-docs';
 
 If you are not using React, you can still embed a calling/chat composite into your html page with a simple `<script>` tag and several lines of code like:
 
+You can easily find prebuilt bundles from:
```

Comment:
Are we not following SemVer in releasing these bundles? Or is `releases/latest

Reply:
Yep, this is alwayls pointing you to the latest beta release